frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDrawerRightContent.tsx

```diff
@@ -25,12 +25,18 @@ const PipelineRunDrawerRightContent: React.FC<PipelineRunDrawerRightContentProps
   parameters,
   onClose,
 }) => {
+  const [, setWidth] = React.useState<number>(0);
+
   if (!task) {
     return null;
   }
 
   return (
     <DrawerPanelContent
+      // This forces a re-render to solve resize of the Log viewer inside Drawer.
+      onResize={(_event: MouseEvent | TouchEvent | React.KeyboardEvent, newWidth: number) => {
+        setWidth(newWidth);
+      }}
```

Comment:
Is this working as expected? The log viewer doesn't seem to be updating any differently while resizing the drawer.

In fact here the lines start to wrap on themselves:
![image](https://github.com/opendatahub-io/odh-dashboard/assets/14068621/37240941-0ff9-4170-8bc6-0a041f3857c2)

Reply:
https://github.com/opendatahub-io/odh-dashboard/pull/2184#discussion_r1440495709
Also there is an PR raised for the same in patternfly logviewer: https://github.com/patternfly/react-log-viewer/pull/28
cc @christianvogt

Reply:
You should followup with PF on the issue or slack to have them look into fixing the issue.